demos/using-dev-build/dfx.json

```diff
@@ -0,0 +1,22 @@
+{
+  "canisters": {
+    "internet_identity": {
+      "type": "custom",
+      "candid": "internet_identity.did",
+      "wasm": "internet_identity.wasm",
+      "build": "curl -sSL https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm -o internet_identity.wasm"
```

Comment:
So, it's official this is the best way to get other canisters, I guess I could do the same for Ledger and Governance, right?

Reply:
I don't know if this is the official best way, but I've created a [ticket](https://github.com/dfinity/sdk/issues/2085) in the SDK repo. Re. Ledger and Governance, yes, I don't see why that wouldn't work!